webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].[chunkhash].js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  optimization: {
+    runtimeChunk: 'single',
+  },
+};
```

Comment:
[OPTIONAL] Well done overall 💯 but there's a configuration *WARNING* ⚠️ because the 'mode' option has not been set so `webpack` will fall back to `production` for this value. 🤔  In order to fix that set the `mode` option to `development` or `production` to enable defaults for each environment. 🌴  You can also set it to `none` to disable default behavior. Learn more: https://webpack.js.org/configuration/mode/ 🔗

Reply:
@didierganthier  Hey Didier thanks a lot for the recommendations. I think I just added it. Hope you give it a review to check if it was well implemented.

Reply:
@didierganthier  Hey Didier thanks a lot for the recommendations. I think I just added it. Hope you give it a review to check if it was well implemented.

Reply:
Well done adding it @jlberbesi  💯 🙂